src/types.d.ts

```diff
@@ -0,0 +1,148 @@
+type Color = [number, number, number, number] | [number, number, number];
```

Comment:
This is probably the type that needs the most help. Is string allowed for all color props as well, or just some?